csrc/host_ir/jit.cpp

```diff
@@ -828,10 +973,54 @@ void HostIrJitImpl::registerExternalFunctions() {
             c10::nullopt);
       });
 
-  // delete a newed tensor
-  void* delete_tensor_func_ptr = reinterpret_cast<void*>(
-      +[](at::Tensor* tensor) -> void { delete tensor; });
+  // launch kernel function
+  void* launch_kernel_func_ptr =
+      reinterpret_cast<void*>(+[](int64_t cache_id,
+                                  at::Tensor** input_tensors,
+                                  int64_t num_inputs,
+                                  at::Tensor** output_tensors,
+                                  int64_t num_outputs,
+                                  void* launch_kernel,
+                                  void* container) {
+        auto* launch_kernel_ptr =
+            static_cast<hir::LaunchKernel*>(launch_kernel);
+        auto* container_ptr =
+            static_cast<hir::HostIrContainer*>(container);
+        KernelArgumentHolder input_args, output_args;
+        input_args.setCacheId(cache_id);
+
+        for (int64_t i = 0; i < num_inputs; i++) {
+          input_args.push(
+              *input_tensors[i]); 
+        }
+        for (int64_t i = 0; i < num_outputs; i++) {
+          output_args.push(
+              *output_tensors[i]); 
+        }
+        input_args.setDeviceIndex();
+        container_ptr->getKernelExecutor(launch_kernel_ptr->groupId())
+            ->run(
+                input_args,
+                output_args,
+                launch_kernel_ptr->launchParams(),
+                launch_kernel_ptr->compileParams());
+      });
 
+  // matmul_out function
+  void* matmul_out_func_ptr = reinterpret_cast<void*>(
+    +[](at::Tensor* t_out, at::Tensor* t_a, at::Tensor* t_b) {
+      at::matmul_out(*t_out, *t_a, *t_b);
+    });
+
+  // linear_out function
+  void* linear_out_func_ptr = reinterpret_cast<void*>(
+    +[](at::Tensor* t_out, at::Tensor* t_in, at::Tensor* t_weight, at::Tensor* t_bias) {
```

Comment:
```suggestion
    +[](at::Tensor* out, at::Tensor* in, at::Tensor* weight, at::Tensor* bias) {
```
Given these variables are `at::Tensor`s and this function is too short, having `t_` in their names doesn't provide extra information.